app/src/main/java/com/example/encoratask/adapter/SimpleItemRecyclerViewAdapter.kt

```diff
@@ -0,0 +1,51 @@
+package com.example.encoratask.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.example.encoratask.R
+import com.example.encoratask.model.Character
+
+/**
+ * Created by Erick Sanchez
+ * Revision 1 - 10/06/21
+ */
+open class SimpleItemRecyclerViewAdapter(private val values: ArrayList<Character>) :
```

Comment:
ArrayList is an implementation of List. If you want to have a clean code, you should depend on interfaces instead of implementations

Reply:
I understand :) but I think its good practice use ArrayList to

https://stackoverflow.com/questions/2279030/type-list-vs-type-arraylist-in-java